pandas/tests/io/test_parquet.py

```diff
@@ -252,6 +301,10 @@ def test_duplicate_columns(self, fp):
 
     def test_unsupported(self, fp):
 
+        # bool with none
+        df = pd.DataFrame({'a': [True, None, False]})
```

Comment:
its just how it is now in pandas. eventually this will be done 'correctly'. I just want to note the differences. Categorical are very efficient at this structure, and if pandas2 weren't coming along I would change it, but leaving for now.